tools/update-llhttp.sh

```diff
@@ -1,10 +1,11 @@
-#!/bin/sh
-set -e
-# Shell script to update nghttp2 in the source treee to specific version
+#!/bin/bash
```

Comment:
The markdown linter require to use `bash` in our docs, but the sh linter require we use `sh` in our scripts.
```suggestion
#!/bin/sh
```